independent-publisher-connectors/Udemy/apiDefinition.swagger.json

```diff
@@ -0,0 +1,828 @@
+                        "description": "id"
+                      },
+                      "title": {
+                        "type": "string",
+                        "description": "title"
+                      },
+                      "url": {
+                        "type": "string",
+                        "description": "url"
+                      },
+                      "is_paid": {
+                        "type": "boolean",
+                        "description": "is_paid"
+                      },
+                      "price": {
+                        "type": "string",
+                        "description": "price"
+                      },
+                      "price_detail": {
+                        "type": "string",
+                        "description": "price_detail"
+                      },
+                      "price_serve_tracking_id": {
+                        "type": "string",
+                        "description": "price_serve_tracking_id"
+                      },
+                      "visible_instructors": {
+                        "type": "array",
+                        "items": {
+                          "type": "object",
+                          "properties": {
+                            "_class": {
+                              "type": "string",
+                              "description": "_class"
+                            },
+                            "title": {
+                              "type": "string",
+                              "description": "title"
+                            },
+                            "name": {
+                              "type": "string",
+                              "description": "name"
+                            },
+                            "display_name": {
+                              "type": "string",
+                              "description": "display_name"
+                            },
+                            "job_title": {
+                              "type": "string",
+                              "description": "job_title"
+                            },
+                            "image_50x50": {
+                              "type": "string",
+                              "description": "image_50x50"
+                            },
+                            "image_100x100": {
+                              "type": "string",
+                              "description": "image_100x100"
+                            },
+                            "initials": {
+                              "type": "string",
+                              "description": "initials"
+                            },
+                            "url": {
+                              "type": "string",
+                              "description": "url"
+                            }
+                          }
+                        },
+                        "description": "visible_instructors"
```

Comment:
@nanddeepn All the description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)